cookbook/recipes/configure.rb

```diff
@@ -0,0 +1,43 @@
+template "#{node[:singularity][:conf_dir]}/singularity.yaml" do
+  source 'singularity.yaml.erb'
+  owner  'root'
+  group  'root'
+  mode   0644
+  variables(baragon_url:
+    'http://localhost:' \
+    "#{node['baragon']['service_yaml']['server']['connector']['port']}" \
+    '/baragon/v2')
```

Comment:
This is another situation where this setting makes sense in a testing / exploratory environment, but it's not a safe assumption that you'd be running BaragonService on all the same hosts as SingularityService in a production environment. I think a good compromise would be to make this an overridable setting.